docfx_project/ziti/quickstarts/network/common-quickstart.md

```diff
@@ -8,7 +8,7 @@ When first launched - the instace will deposit a file into the file system at
 > changes (such as a system reboot) the image needs to be reconfigured becuase the certificates will no longer be valid.
 > This file is used to reconfigure the system in this event and it happens automatically on startup.
 
-Now, ssh to the newly created machine. Once there you can obtain the username and password for your
+Now, ssh to the newly created machine. Once there you can obtain the username (defaults to "admin") and password for your

```

Comment:
fyi. in case it ever changes i didn't specify the default user name on purpose.

Reply:
You're right it makes more sense to be prescriptive here. I'll add a step to capture or make a note of the username in addition to the password.